app/controllers/subscriptions_controller.rb

```diff
@@ -28,13 +28,18 @@ def create
     render json: { id: subscription.id }, status: status
   end
 
-  def change_frequency
+  def show
+    subscription = Subscription.find(subscription_params.require(:id))
+    render json: subscription.to_json
```

Comment:
I think we should aim for consistency here, since `update` returns `render json: { subscription: subscription }, status: :ok`. I'm happy with either one but they should be the same :)

Reply:
Ah yep, I agree. I'll make it return everything under a `subscription` key like `update`.